assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -511,6 +532,22 @@ const baseResolvers = {
 			}
 		}
 	},
+
+	*hasModuleAccess( slug ) {
+		const registry = yield Data.commonActions.getRegistry();
+
+		const existingCheckAccess = registry
+			.select( CORE_MODULES )
+			.hasModuleAccess( slug );
+
+		if ( existingCheckAccess ) {
```

Comment:
We need to return only when the `existingCheckAccess` is `undefined` because if it is `false` then it means that we have already fetched access information and we don't need to send it one more time.